source/documentation/breaking-changes/bogus-combinators.liquid

```diff
@@ -0,0 +1,67 @@
+---
+title: "Breaking Change: Invalid Combinators"
+introduction: >
+  Sass has historically been very permissive about the use of leading, trailing,
+  and repeated combinators in selectors. These combinators are being deprecated
+  except where they're useful for nesting.
+---
+
+{% markdown %}
+Sass has historically supported three invalid uses of combinators:
+
+* Leading combinators, as in `+ .error {color: red}`.
+
+* Trailing combinators, as in `.error + {color: red}`.
+
+* Repeated combinators, as in `div > > .error {color: red}`.
+
+None of these are valid CSS, and all of them will cause browsers to ignore the
+style rule in question. Supporting them added a substantial amount of complexity
+to Sass's implementation, and made it particularly difficult to fix various bugs
+related to the `@extend` rule. As such, we [made the decision] to remove support
+for these uses.
+
+[made the decision]: https://github.com/sass/sass/issues/3340
+
+**There is one major exception**: leading and trailing combinators may still be
+used for nesting purposes. For example, the following is still very much
+supported:
+{% endmarkdown %}
+
+{% codeExample 1 %}
```

Comment:
The code examples are broken (the tabs don't change on click). I see a JS error: `figure.tabs is not a function`. Wanted to ask if there's a simple fix before looking too much into it.

Reply:
@jerivas Are you still seeing this? I can't reproduce (after merging all the other PRs in here); code examples are working for me locally.

Reply:
Just checked, seems to be fixed 🎉